tools/cli/commands/changelog.js

```diff
@@ -743,6 +743,17 @@ async function promptChangelog( argv, needChangelog, types ) {
 		},
 	} );
 
+	let userFacing = false;
+	// Ask if this is a user facing change if we detect Jetpack.
+	if ( argv.project === 'plugins/jetpack' ) {
+		userFacing = await prompt( {
+			type: 'confirm',
+			name: 'userFacing',
+			message: 'Is this a Jetpack change that site admins would like to know about? (Y/n)',
+			default: 'true',
```

Comment:
When I was working on the other PR, I noted that for 'confirm' it renders better if you do like this instead.
```suggestion
			initial: true,
```
That lets you skip having to manually include "(Y/n)" in the message field.